MIP/mip-21/README.md

```diff
@@ -0,0 +1,70 @@
+
+## Specification
+
+The Cross-Chain Transfer Protocol (CCTP) operates using a set of attestors that validate all bridge transactions occurring between chains. These attestors listen for bridge events, confirm the validity of transactions, and, if necessary, revert them by providing appropriate proofs. A transfer allowance managed by a centralized authority further ensures that breaches are mitigated by limiting the amount of value transferred.
+
+To align with this approach and reduce complexity:
+
+- **Bridge Relayer as Attestor**: The Bridge Relayer in our system will function as an attestor, operating via a multisig. This relayer, through aggregated signatures, produces a proof on a signed message.
+  
+- **User Interaction**: Users, or other parties interested in completing the bridge, retrieve their funds on the target chain by submitting the signature and message.
+
+- **Validation Mechanism**: The off-chain signature produced by the multisig is verified against the message on-chain. The challenge is to make sure that the signature is only used once, but the beauty is that the signature is comprised of a multi-party system that do not need to provide onchain signatures. But relying on a protocol like CCTP that has been tested in live environments significantly reduces the risk of any unexpected utilization of off-chain and on-chain proofs.
+
+Or on Circle's own words:
+
+1. An on-chain component on source domain emits a message.
+2. Circle's off-chain attestation service signs the message.
+3. The on-chain component at the destination domain receives the message, and forwards the message body to the specified recipient.
+
+This approach minimizes the need for additional components and avoids the reliance on synthetic assets, leading to a simplified and cost-effective bridge solution.
+
+## Reference Implementation
+
+For more details on how this model operates, refer to USDC's Cross-Chain Transfer Protocol (CCTP) documentation:  
+[USDC CCTP Overview](https://developers.circle.com/stablecoins/docs/generic-message-passing)
+
+## Verification
+
+### 1. **Correctness**: 
+The correctness of this proposal can be established by leveraging the success of CCTP, which has been operational and widely adopted.
+
+### 2. **Security Implications**:
+The use of a multisig setup adds a layer of security, with aggregated signatures preventing unauthorized access. The transfer allowance mechanism further mitigates potential breaches by limiting how much can be transferred.
+
+### 3. **Performance Impacts**:
+Because we would use a multisig off-chain signature to complete bridges, the set of attestors and the multisig address to produce the signature that we have to check the message against it could be the same set of the L2 Settlement signers.
```

Comment:
```suggestion
Because we would use a multisig off-chain signature to complete bridges, the set of attestors for the bridge and the multisig address to produce the signature that we have to check the message against, could be the same set as the L2 Settlement attestors.
```

Reply:
Is this what is meant? Are you connecting to our FFS attestor solution?

Reply:
It could be, not necessarily initially.